src/features.rs

```diff
@@ -32,9 +32,16 @@ fn try_find() -> Result<Vec<String>, Ignored> {
     // The hash at the end is ascii so not lossy, rest of conversion doesn't
     // matter.
     let test_binary_lossy = test_binary.to_string_lossy();
-    let hash = test_binary_lossy
-        .get(test_binary_lossy.len() - 17..)
-        .ok_or(Ignored)?;
+    let hash = if cfg!(windows) {
```

Comment:
This will break if someone is cross-compiling tests. Do we care?

Reply:
Nope, don't care. They can send a fix if it comes up. Thanks for flagging though!